locale/de/about/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: about.hbs
+title: Über Node.js
+trademark: Trademark
+---
+# Über Node.js&reg;
+
+Als asynchrone, Event-basierte Laufzeitumgebung wurde Node speziell für die
+Entwicklung von skalierbaren Netzwerkanwendungen entworfen. Im nachfolgenden
+"Hallo Welt" Beispiel können viele Verbindungen gleichzeitig bearbeitet werden.
+Bei jeder neuen Verbindung wird die Rückruffunktion ausgeführt. Gibt es jedoch
+nichts zu tun, befindet sich Node im Ruhezustand.
+
+```javascript
+const http = require('http');
+
+const hostname = '127.0.0.1';
+const port = 3000;
+
+const server = http.createServer((req, res) => {
+  res.statusCode = 200;
+  res.setHeader('Content-Type', 'text/plain');
+  res.end('Hallo Welt\n');
+});
+
+server.listen(port, hostname, () => {
+  console.log(`Server läuft unter http://${hostname}:${port}/`);
+});
+```
+
+Dies steht im Gegensatz zu den heutzutage üblichen Modellen für Nebenläufigkeit,
+bei denen Threads des Betriebssystems genutzt werden. Thread-basiertes
+Networking is vergleichsweise ineffizent and sehr schwer umzusetzen.
+Zudem müssen sich Node Nutzer nicht um Deadlocks im Prozess sorgen, da es
+keine Blockierung gibt. Fast keine Funktion in Node führt direkt I/O
+Operationen aus, daher wird der Prozess nie blockiert. Da nichts blockiert,
+können mit Node sinnvoll skalierbare Systeme entwickelt werden.
+
+Wenn einige dieser Konzepte unbekannt sind, gibt es hier einen Artikel zum Thema
+[blockierend vs. blockierungsfrei][] (auf Englisch).
+
+---
+
+Node ähnelt im Design und ist beeinflusst von Systemen wie Rubys
+[Event Machine][] oder Pythons [Twisted][]. Node führt das Event Modell noch
```

Comment:
Vorschlag: Ich würde zur besseren Lesbarkeit die Links in Anführungszeichen setzen, weil es sich um Eigennamen handelt:

Rubys "[Event Machine][]" oder Pythons "[Twisted][]"